src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -106,6 +106,11 @@ const terminalConfiguration: IConfigurationNode = {
 			default: 'right',
 			description: localize('terminal.integrated.tabs.location', "Controls the location of the terminal tabs, either to the left or right of the actual terminal(s).")
 		},
+		[TerminalSettingId.TabFocusMode]: {
+			description: localize('tabFocusMode', "Controls whether the terminal receives tabs or defers them to the workbench for navigation."),
```

Comment:
Let's mention how this is a special override and link to the main tab focus mode setting